src/Controls/src/Core/Menu/MenuFlyoutItem.cs

```diff
@@ -1,12 +1,22 @@
 using System;
 using System.Collections.Generic;
+using System.Collections.ObjectModel;
 using System.ComponentModel;
 using System.Windows.Input;
 using Microsoft.Maui.Controls.StyleSheets;
 
 namespace Microsoft.Maui.Controls
 {
-	public partial class MenuFlyoutItem : MenuItem, IMenuFlyoutItem
-	{
-	}
+    public partial class MenuFlyoutItem : MenuItem, IMenuFlyoutItem
+    {
+		public MenuFlyoutItem()
+		{
+			KeyboardAccelerators = new ObservableCollection<KeyboardAccelerator>();
+		}
+
+		public IList<KeyboardAccelerator> KeyboardAccelerators { get; }
```

Comment:
Should this be a bindable property?

Reply:
@PureWeen thoughts? I remember you suggesting not, but I could be misremembering

Reply:
We don't seem to be consistent here. Looking around we don't have a lot of places where we've made list structures bindable.  It's also just easier to create this as a collection we own and know implements INCC for now. Otherwise, we'll need to worry about monitoring the type of collection as well. If someone wants to bind to this, they can always use a converter to translate.